vendor/models/price.py

```diff
@@ -1,6 +1,7 @@
 from django.conf import settings
 from django.db import models
-from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Can you confirm that ugettext_lazy() works in this case and does not create any "promise" conflicts.

Reply:
@mgcamisa changed it because apparently the ugettext was not working correctly.